### PR TITLE
Feat/improvements mentioned in #35

### PR DESCRIPTION
**Pull request by jnnkls**
Improvements mentioned in #35 

* [x] We could give the footer a darker color so that the contrast is more visible.

* [x] Then we could reduce the copyright text a little bit (e.g. FHA Bereich 5) because on mobile it would otherwise be a bit squeezed.

* [x] On certain screen sizes (e.g. iPhone 6) I get interesting white/greenish stripes at the lower wave. In addition, the footer slips a bit too high.
stripes2

* [x] I don't know if you use a custom font but on my browser the text FortuneTeller shows as Comic Sans. I would be happy if we could change that 😉.

* [ ] On mobile phones bevor iPhone 6 / Samsung Galaxy 6 the upper part (Anrede) starts squeezing into the header.

### CODE REVIEW

File: web/src/components/wave/Wave.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Wave<P extends WaveProps, S extends WaveState> extends React.Component<P,
     let cTop = this.props.colorTop
     let cBottom = this.props.colorBottom
     if (this.props.rotate) {
-      state.style.transform = "rotateZ(180deg)"
+      state.style.transform = "rotateX(180deg) translate(0, -1px)"
```
Comment (KnutZuidema) [inline web/src/components/wave/Wave.tsx:32]: And this is why I dislike frontend development :joy: Thanks

File: web/src/config.ts [hunk 2]
```diff
@@ -3,5 +3,6 @@ export let config = {
   sourceUrl: "https://github.com/FHA-FB5/FortuneTeller",
   projectName: "Fortune Teller",
   copyrightUrl: "https://fsr5.de",
-  copyrightOwner: "FH Aachen Fachbereich 5"
+  copyrightOwner: "FH Aachen Fachbereich 5",
+  copyrightOwnerShort: "FHA Bereich 5"
```
Comment (KnutZuidema) [inline web/src/config.ts]: `Bereich` does not really convey the same meaning IMO. I would suggest this: `FH Aachen FB 5`

**Review by KnutZuidema — CHANGES_REQUESTED** [review]
Looks good, just the copyright text :+1:

File: web/src/config.ts (hunk 2 above)
Comment (jnnkls) [reply web/src/config.ts]: 👍

File: web/src/components/wave/Wave.tsx (hunk 1 above)
Comment (jnnkls) [reply web/src/components/wave/Wave.tsx:32]: no problem 😉

**Review by KnutZuidema — APPROVED** [review]
Thanks :+1:

**Pull request merged**